recipes/json-c/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class JSONCConan(ConanFile):
+    name = "json-c"
+    description = "JSON-C - A JSON implementation in C"
+    topics = ("conan", "json-c", "json", "encoding", "decoding", "manipulation")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/json-c/json-c"
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt", "json-c.patch"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        'shared': False,
+        'fPIC': True
+    }
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.name + "-" + self.version + '-20180305'
```

Comment:
```suggestion
        url = self.conan_data["sources"][self.version]["url"]
        extracted_dir = url[url.rfind('/')+1:url.find(".tar.gz")]
```
`20180305` should not be hardcoded.

Reply:
Good idea, thanks!